### ISSUE TITLE
Upgrade to Electron 31

### ISSUE THREAD

**Issue by achou11** [digidem/comapeo-desktop#23]
At the time of writing, Electron [31.1.0](https://releases.electronjs.org/release/v31.1.0) is available. I was going to do this work when the v31 line was initially released, but wanted to wait on better-sqlite3 to publish a release that included compatible prebuilds. Starting from [v11.1.0](https://github.com/WiseLibs/better-sqlite3/releases/tag/v11.1.0), better-sqlite3 now includes the compatible prebuilds, which makes this work more reasonable to do.

Upgrading should (hopefully) not be problematic in terms of existing app code. Better to do this sooner rather than later, when there's more app code that's reliant on Electron APIs.

**Comment by achou11** [comment]
No longer relevant, as we're currently on a more recent Electron version than what's stated here
